kubenow/gluster-s3/templates/gluster-s3_pvc_main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .Values.pvc1Name }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: default
```

Comment:
we may default to something else than gluster in the future, can you se this to the gluster fs storage class? @andersla can point you there. Please do the same for the other PVCs as well.